learning/dashboard/Group-5/OPSG5.md

```diff
@@ -0,0 +1,61 @@
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type Angular Snippets and click on the first option and click install and wait for it to install.
+![image](https://user-images.githubusercontent.com/123230201/227721915-a0e28d2c-f0ca-44a8-98cb-f3a7df7318ab.png) 
+#
+# How to install JSLint extension in VS Code.
+
+## After you've finished installing the Angular Snippets extension on VS code, the next step is to install the JSLint extension. In order to download the extension VS Code, we need to do the following steps.
+
+## 1. Open VS Code and click on the extension button of the left side of VS Code. 
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type JSLint and click on first option and click install and wait for it to install. 
+![image](https://user-images.githubusercontent.com/123230201/227722349-5c408e0e-188e-40fc-8cd1-08530beeac9a.png) <p> 
+#
+# How to install JSON extension in VS Code 
+
+## After you've finsihed installing the JSLint extension on VS Code, the next step is to install the JSON extension. In order to download the extension VS Code, we need to do the following steps. 
+
+## 1. Open VS Code and click on the enxtension button of the left side of VS Code. 
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type JSON and click on the first option and click install and wait for it to install. 
+![image](https://user-images.githubusercontent.com/123230201/227724297-64c61345-83e1-4e33-a7b9-0ee8a7f4a8b2.png)
+#
+# How to install Live Server extension in VS Code 
+
+## After you've finished installing the JSON extension on VS Code, the next step is to install the Live Server extension. In order to download the extension VS Code, we need to do the following steps. 
+
+## 1. Open VS Code and click on the extension button of the left side of VS Code.
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type Live Server and click on the first option and click install and wait for it to install. 
+![image](https://user-images.githubusercontent.com/123230201/227724718-2e98ef5c-9d77-4c27-b256-dd16b9b5c71a.png)
+# 
+# How to install Prettifier extension in VS Code 
+
+## After you've finsihed installing the Live Server extension on VS Code, the next step is to install the Prettifier extension. In order to download the extension VS Code, we need to do the following steps. 
+
+## 1. Open VS Code and click on the extension button of the left side of VS Code. 
+![image](https://user-images.githubusercontent.com/123230201/227721833-f5057b9e-7113-435c-add3-8ffdc125c82d.png)
+
+## 2. Type Prettifier extension and click on the third option and click install and wait for it to install. 

```

Comment:
This seems to be the incorrect Prettifier extension. This one is downloaded some 123 times.
The correct prettifier extension is downloaded many more times.
Consider adding some usage instructions and how it helps.